src/kanata/mod.rs

```diff
@@ -1620,6 +1654,8 @@ impl Kanata {
                     }
                 }
             }
+            #[cfg(all(target_os = "windows", feature = "gui"))]
+            gui_tx.notice();
```

Comment:
There seems to be a lot of code added just to get these `.notice()` changes. Perhaps it is better to store `NoticeSender` in a `static` `OnceLock` or `Mutex` in another module and call into that from here to reduce code changes, instead of passing it down through all the calls.

Reply:
Updated, thought I thought explicit vars were more Rusty vs. globals